frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -178,6 +260,47 @@ export const surveyLogic = kea<surveyLogicType>([
                 editingSurvey: (_, { editing }) => editing,
             },
         ],
+        isEditingQuestion: [
```

Comment:
you don't need these reducers anymore

Reply:
same for the actions for setting this value!

Reply:
Indeed!